x-pack/platform/plugins/shared/alerting/server/lib/rule_gaps/soft_delete/soft_delete_gaps.ts

```diff
@@ -62,6 +62,6 @@ export const softDeleteGaps = async (params: SoftDeleteGapsParams) => {
       throw new Error('Some gaps failed to soft delete');
     }
   } catch (e) {
-    logger.error(`Failed to soft delete gaps for rule ${ruleId}: ${e.message}`);
+    logger.error(`Failed to soft delete gaps for rules ${ruleIds.join(', ')}: ${e.message}`);
```

Comment:
Should I print the count of rule ids? I am thinking it is not wise to print the whole list if it is too big